lms/templates/ux/reference/teams-base.html

```diff
@@ -0,0 +1,165 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
NIT: our [new file standards](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-General) prefer a 4 space (soft tab) indentation. Not sure if you want to sync that up here or wait till the PL is in this app. Your call.